src/odemis/driver/actuator.py

```diff
@@ -1006,17 +1006,16 @@ def __init__(self, name, role, dependencies, axis_name, offset=0,
         logging.debug("Subscribing to position of dependency %s", dep.name)
         dep.position.subscribe(self._update_dep_position, init=True)
 
-        referenced = False
         if model.hasVA(dep, "referenced") and axis_name in dep.referenced.value:
             referenced = dep.referenced.value[axis_name]
             self.referenced = model.VigilantAttribute({self._axis: referenced}, readonly=True)
             dep.referenced.subscribe(self._update_dep_ref)
-        if not referenced:
-            # The initialisation will not fail if the referencing fails
-            f = self.reference({axis})
-            f.add_done_callback(self._on_referenced)
-        else:
-            self.moveAbs({self._axis: self._dependency.position.value[self._caxis]}).result()
```

Comment:
Are you sure we can drop this completely? Shouldn't we try to move to the right position instead, including the offset?

Reply:
Here the "right" position should be the "current" position, so in general no move should happen. 

We could argue on what happens if the current position is out of the range. Should we explicitly request a move to reach the nearest range? In general, at this level (of abstraction), I think it's preferable to never automatically move. This is the initialisation code of (a wrapper of) a hardware component. It's can be really dangerous do decide that because the back-end starts, the hardware has to move. So far we *only* do a move if it's a referencing move and the user indicated it's fine in the arguments (ie, YAML file). Up to the higher level (eg, the Optical path manager) to decide where to move things.

Reply:
ok